tests/hawtio-test-suite/pom.xml

```diff
@@ -22,6 +22,21 @@
     <xtf.version>0.31</xtf.version>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpcore</artifactId>
+        <version>${httpcore-version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpclient</artifactId>
+        <version>${httpclient-version}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
```

Comment:
what's the motivation behind this?

Reply:
Without this, we have different versions of Http Client 4 visible in IDE (debugging problems):
![image](https://github.com/hawtio/hawtio/assets/306565/943432ef-1bc4-4360-9d15-a169a002b833)

Adding managed dependencies gives me:
![image](https://github.com/hawtio/hawtio/assets/306565/6151b92c-e3ac-4778-b4e8-76a45b7098fd)

Wrong (old) versions of httpclient/httpcore come from:
```
+- com.github.docker-java:docker-java:jar:3.3.4:compile
|  +- com.github.docker-java:docker-java-core:jar:3.3.4:compile
...
|  +- com.github.docker-java:docker-java-transport-jersey:jar:3.3.4:compile
...
|  |  +- org.apache.httpcomponents:httpcore:jar:4.4.13:compile
|  |  +- org.apache.httpcomponents:httpclient:jar:4.5.12:compile
```